linkerd/addr/fuzz/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "linkerd-addr-fuzz"
+version = "0.0.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+publish = false
+edition = "2018"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = "0.4"
+
+[dependencies.linkerd-addr]
+path = ".."
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
+
+[features]
+fuzzing = []
```

Comment:
the fuzz target is a separate crate that's only compiled when fuzzing, correct? why do we also need a feature flag to enable it?

Reply:
The reason we do this is because otherwise `cargo check` fails due to the use of `cfg(fuzzing)` wrapped around the fuzz modules. See here: https://github.com/linkerd/linkerd2-proxy/pull/961#discussion_r605010415

Reply:
Ah, okay, I see, that makes sense. I think the use of the feature flag is probably appropriate in this case.